src/build.d

```diff
@@ -241,19 +241,27 @@ version(Windows)
     // Build the msvc-dmc compiler wrapper
     auto buildMsvcDmc()
     {
+        enum targetName = "msvc-dmc";
+
         Dependency dependency = {
-            target: env["G"].buildPath("msvc-dmc").exeName,
-            sources: [`vcbuild\msvc-dmc`],
+            target: env["G"].buildPath(targetName).exeName,
+            sources: [`vcbuild\` ~ targetName],
+            name: "Building " ~ targetName,
```

Comment:
Not saying that it's great, but the convention which was taken over from the Makefile is to do sth. like:

```d
name: "(CC) VCBUILD " ~ targetName
```

Though we probably should start using `DC` wherever a D compiler is used.